CONTRIBUTING.md

```diff
@@ -38,7 +38,7 @@ detailed guidance about how to contribute to the project.
 #### Documentation
 
 Libra's developer website is also open source (the code can be found in this
-[repository](https://github.com/libra/developers.libra.org/website/)).  It is built using
+[repository](https://github.com/libra/libra/developers.libra.org/website/)).  It is built using
 [Docusaurus](https://docusaurus.io/):
 
 If you know Markdown, you can already contribute! How to contribute can be found in the [website
```

Comment:
This link needs to be updated too

Reply:
I would just remove the second sentence here starting with "How to contribute...."